vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -34,6 +34,7 @@ body.mobile .initials-images .initials-image,
 body.tablet .initials-images .initials-image {
     height: auto;
     max-width: 600px;
+    object-fit: contain;

```

Comment:
This is a breaking change that is risky as hell, as this component is used in several scenarios.

Just use the existing `imageObjectFit` prop wherever you needed this for.